apps/cowswap-frontend/src/modules/tokensList/pure/TokenInfo/styled.ts

```
@@ -1,6 +1,7 @@
 import { Media } from '@cowprotocol/ui'
 
 import styled from 'styled-components/macro'
+import { LinkStyledButton } from 'theme'
```

ESLint

[error] 'LinkStyledButton' is defined but never used.

(unused-imports/no-unused-imports)